docs/html/ux-research-design/contribute.md

```diff
@@ -0,0 +1,24 @@
+# How to Contribute
+
+## Participate in UX Research
+
+It is important that we hear from pip users so that we can:
+
+- Understand how pip is currently used by the Python community
+- Understand how pip users _need_ pip to behave
+- Understand how pip users _would like_ pip to behave
+- Understand pip’s strengths and shortcomings
+- Make useful design recommendations for improving pip
+
+If you are interested in participating in pip user research, please [join pip’s user panel](https://mail.python.org/mailman3/lists/pip-ux-studies.python.org/).
```

Comment:
Is the mailing list still active? I notice that the link on that page for more information is broken.

Reply:
It is. I'll fix the link on the mailing list description. :)